tsconfig.json

```diff
@@ -20,5 +20,6 @@
     "noUnusedParameters": true,
     "noFallthroughCasesInSwitch": true
   },
-  "include": ["src"]
+  "include": ["src/**/*.ts", "src/**/*.d.ts", "src/**/*.tsx", "src/**/*.vue"],
```

Comment:
We don't need .tax do we?

Reply:
you're right, this is just copy pasta from new vite project